plugins/resources/unixfilesystem/src/irods_get_detached_mode_vault_path.cpp

```diff
@@ -0,0 +1,80 @@
+#include "irods/private/irods_get_detached_mode_vault_path.hpp"
+
+#include "irods/filesystem/path.hpp"
+#include "irods/irods_error.hpp"
+#include "irods/irods_logger.hpp"
+#include "irods/irods_lookup_table.hpp"
+#include "irods/irods_string_tokenize.hpp"
+
+#include <vector>
+#include <sstream>
+#include <string>
+#include <tuple>
+
+const std::string HOST_LIST("host_list");
+
+// returns a tuple with the following:
+//   bool - true iff the resource_hostname is in HOST_LIST string
+//   std::optional<std::string> - the new vault path if it is defined
+//
+//   Note:  There is a forward declaration of this in unit_tests/src/test_host_list_context_string.cpp
```

Comment:
Not needed any more since we have a header for this now

Reply:
Yes, it is no longer there.  I just forgot to remove the comment.  Will remove.